### ISSUE TITLE
Server Broke I Too many Files Error

### ISSUE THREAD

**Issue by TestingUser661** [TheLimeGlass/Skungee#51]
https://hastebin.com/hacubefeju.shell

I have 0 scripts in globalscripts.
Using Skungee latest
Java 8
Linux (ubuntu)

Using only 2 Spigots and 1 Bungee (after a certain amount of time, it said that error)
then I weren't able to use the server, connect to it it said "Disconnected", and in Bungee console it showed errors that it took a long time to send the MOTD message. 

It might have to do with the unknown host name error but i'm not 100% sure.

**Comment by TestingUser661** [comment]
Update: It was spamming it constantly in the Lobby (spigot) server, It did not stop until i restarted

**Comment by TheLimeGlass** [comment]
Posting this here incase hastebin expires https://pastebin.com/XCsCt1TJ

**Comment by TheLimeGlass** [comment]
If you have other issues please share those console errors aswell. The too many files error happens when the Socket aka Skungee is connecting and disconnecting frequently, so something either the server or another plugin is causing this.

**Comment by TestingUser661** [comment]
Yup, and in bungee it says this: https://pastebin.com/raw/BdFYWEF8.
Basically the server won't work.

When can you fix this? It is a very bad bug affecting my players + server

**Comment by TheLimeGlass** [comment]
You need to configure your security settings to match the exact from the Spigot Skungee, they're currently not the same and it's erroring. Also don't just directly copy the entire configuration from Spigot to Bungee Skungee as I have seen users do that in the past.

**Comment by TestingUser661** [comment]
Yes, I don't know if it is related to the bug above. But I did copy and paste the configs from the #1 spigot to the other spigots, and this happens sometimes.

**Comment by TheLimeGlass** [comment]
Don't copy a paste the configurations.

**Comment by TestingUser661** [comment]
16:31:11 [WARNING] Event ProxyPingEvent(connection=[/ip] <-> InitialHandler, response=ServerPing(version=ServerPing.Protocol(name= , protocol=404), players=ServerPing.Players(max=2019, online=0, sample=null), description=test, modinfo=ServerPing.ModInfo(type=FML, modList=[]))) took 10,011,000,227ns to process!


Usually this error shows in Bungee and players can't get the MOTD, when the above error happens (the post).

**Comment by TheLimeGlass** [comment]
Try to increase the File Descriptor Limit if you own the dedicated server.
